python/pyspark/context.py

```diff
@@ -180,20 +205,20 @@ def __init__(
 
     def _do_init(
         self,
-        master,
-        appName,
-        sparkHome,
-        pyFiles,
-        environment,
-        batchSize,
-        serializer,
-        conf,
-        jsc,
-        profiler_cls,
-    ):
+        master: Optional[str],
+        appName: Optional[str],
+        sparkHome: Optional[str],
+        pyFiles: Optional[List[str]],
+        environment: Optional[Dict[str, str]],
+        batchSize: int,
+        serializer: Serializer,
+        conf: Optional[SparkConf],
+        jsc: JavaObject,
+        profiler_cls: type,
+    ) -> Any:
```

Comment:
`_do_init` is used only for side effects and returns nothing, so should be `-> None`.